ohapi/cassettes/test_update_data_valid_master_access_token.yaml

```diff
@@ -0,0 +1,52 @@
+    headers:
+      Allow: ['GET, HEAD, OPTIONS']
+      Cache-Control: ['must-revalidate, no-store, no-cache, max-age=0']
+      Connection: [close]
+      Content-Language: [en]
+      Content-Type: [application/json]
+      Date: ['Mon, 19 Mar 2018 05:53:05 GMT']
+      Expires: ['Mon, 19 Mar 2018 05:53:06 GMT']
+      Last-Modified: ['Mon, 19 Mar 2018 05:53:06 GMT']
+      Server: [Cowboy]
+      Vary: ['Accept, Accept-Language, Cookie']
+      Via: [1.1 vegur]
+      X-Frame-Options: [SAMEORIGIN]
+    status: {code: 200, message: OK}
+- request:
```

Comment:
It seems like the same request is in this yaml file 2 times, is this intentional?

Reply:
@mcescalante, The function which is being tested is called in the constructor of the class and I am not able to extract any response from it for the first time. Thus, I have to explicitly call the function for the second time to actually test it. Is there any other way around?

Reply:
OK, this "double" makes sense then, and is OK to leave for now.